examples/direct_fidelity_estimation.py

```diff
@@ -95,6 +96,20 @@ def compute_characteristic_function(circuit: cirq.Circuit,
     return sigma_i
 
 
+@dataclass
+class PauliTrace:
+    """
+    A class that contains the Pauli states as described on page 2 of:
+    https://arxiv.org/abs/1104.3835
+    """
+    # Pauli string.
+    P_i: cirq.PauliString
+    # Defined at bottom of left column of page 2.
```

Comment:
Definition of rho_i is simple enough to include inline in addition to the reference.

(Applies to Pr_i and sigma_i below, too.)

Reply:
Done